extension/intents/forms/forms.js

```diff
@@ -96,14 +84,20 @@ intentRunner.registerIntent({
       e.displayMessage = "No text selected";
       throw e;
     }
-    const url = createTabGoogleLucky(selection.text);
+    const newTab = await context.createTabGoogleLucky(selection.text, {
+      hide: true,
+    });
+    const url = newTab.url;
+    const title = newTab.title;
```

Comment:
It just occurred to me that this should actually keep `selection.text` as the text of the link.